### PR TITLE
Added Hello, World! example to Python, Rust, Javascript and C++

### PR DESCRIPTION
**Pull request by james-beans**
I have added a Hello, World! example to all of the programming languages that I know have a printing function. This also might be a bit to beginners like for what you have already put on the languages.

Please tell me if I have messed something up. I think this should all be correct and working.

### CODE REVIEW

**Review by wneel — APPROVED** [review]
I would maybe change the category name to something like "basics" or just "Hello World" but this is a great idea!

+ Idk if this is wanted or not but you just put a line break in the .cpp!

**Comment by james-beans** [comment]
> * Idk if this is wanted or not but you just put a line break in the .cpp!

Yeah if you were talking about under the import of the cpp implementation then yes there is supposed to be an empty line.

**Comment by dostonnabotov** [comment]
Hey, there. Thanks for the contribution. 

Regarding your idea, I like it. I think we can change the category to "Basics" and include basic and handy code snippets in programming languages that doesn't belong to any category.  Also, designed for those who are new to that programming language.

It would also keep it consistent across other languages as well. I'm open to listening more ideas as well.

**Comment by james-beans** [comment]
> Regarding your idea, I like it. I think we can change the category to "Basics" and include basic and handy code snippets in programming languages that doesn't belong to any category. Also, designed for those who are new to that programming language.

I think that having it just called 'Basics' is a bit more general vs what it actually does which is terminal output. This could make it a bit harder to organize snippets without another type of category that goes under another category (sub-categories). Adding a search feature for tags would be nice to have especially for snippets if we are going to make it more general.

**Comment by Mathys-Gasnier** [comment]
One idea would be to name the category "Beginners" just showing the minimal snippet of code needed to print an `Hello World`.
But calling it "Basics" is ok in my books too.

File: public/data/rust.json [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+            "fn main() { // Defines the main running function",
+            "    println!(\"Hello, World!\"); // Prints Hello, World! to the terminal.",
+            "}"
+          ],
+          "tags": ["rust", "printing", "hello-world", "utility"],
+          "author": "James-Beans"
+        }
+      ]
+    },
     "categoryName": "String Manipulation",
```
[the remark this answers is not in the document]
Comment (james-beans) [reply public/data/rust.json:18]: Ok I will fix it!

**Comment by james-beans** [comment]
I added the fix for it to my forked version of this repo and it used to automatically add the changes but it hasn't seemed to update the latest commit to this? Can someone add it for me?

**Comment by Mathys-Gasnier** [comment]
Did you push ? I can't see the change on your repo so you probably forgot

**Comment by james-beans** [comment]
> Did you push ? I can't see the change on your repo so you probably forgot

**Comment by james-beans** [comment]
I did commit it. It just didn't update for like 2 hours straight on this pr. It has now. Can you check if this is finally done so someone can push it?

**Comment by Mathys-Gasnier** [comment]
Before anything I would like you to changes the categories of the snippets you added, I think we came to a conclusion that a category named "Basics" fits better those snippets.

**Comment by james-beans** [comment]
> Before anything I would like you to changes the categories of the snippets you added, I think we came to a conclusion that a category named "Basics" fits better those snippets.

Ok ill change it. That does kind of sound like a good name now after thinking about it.

**Review by Mathys-Gasnier — APPROVED** [review]

**Comment by james-beans** [comment]
Done it! I might add more later but I will add to another pr instead.

**Pull request merged**